scripts/zones/Arrapago_Remnants/mobs/Archaic_Rampart.lua

```diff
@@ -0,0 +1,110 @@
+-----------------------------------
+-- Area: Archaic Rampart
+--  MOB: Arrapago Remnants
+-----------------------------------
+local ID = require("scripts/zones/Arrapago_Remnants/IDs")
+require("scripts/globals/instance")
+require("scripts/globals/status")
+mixins = {require("scripts/mixins/families/rampart")}
```

Comment:
convention has been mixins first so far in our other files, not sure if there is a reason for it or it just happened that way.

Reply:
Worked as is but i moved it up anyways